### PR TITLE
Assignment-16 Commit

### PR DESCRIPTION
**Pull request by Sagar-Fexle-May2022**

### CODE REVIEW

File: src/classes/ExceptionHandling.cls [hunk 1]
```diff
@@ -0,0 +1,106 @@
+/**
+* Description   - ExceptionHandling
+* Created by    - Sagar Nirwal
+* Created Date  - 10/07/2022
+* Revision log  -  V_1.0 - Created  - Your Name - Date
```
Comment (sushilchoudhary7) [inline src/classes/ExceptionHandling.cls:5]: what is your name and date here?

**Review by sushilchoudhary7 — CHANGES_REQUESTED** [review]
Add MyExtID__c field in your PR.

**Comment by Sagar-Fexle-May2022** [comment]
> Add MyExtID__c field in your PR.

Sir, In package?

**Comment by sushilchoudhary7** [comment]
Yes include this MyExtID__c field in the package and add some account records populating this field and try-catch errors.

**Comment by sushilchoudhary7** [comment]
And MyExtID__c fields metadata field as well.
Work on Insert Parent and child record in single DML

**Comment by sushilchoudhary7** [comment]
Work on the Exception Handling section of the assignment.

**Pull request merged**